### DISCUSSION TITLE
Is panda css production ready?

### DISCUSSION THREAD

**Discussion by AsuraKev** [chakra-ui/panda#1534] [Q&A] [answered]
Is panda css production ready?

**Comment by segunadebayo** [comment, marked as answer]
Panda is largely feature complete for most projects, and ready for production.

Over the next months, we'll push v1

**Reply by omridevk** [reply]
I am using it in production and it works great :) really love it.

**Reply by ryanpark** [reply]
Are you able to share your production site ? please

**Reply by omridevk** [reply]
unfortunately it is internal website, but we'll soon be shipping all our sites with it, then i'll share it.
